lib/active_model/serializer.rb

```diff
@@ -167,5 +167,25 @@ def self.get_serializer_for(klass)
         end
       end
     end
+
+    # Transforms an inclusion hash into an array of corresponding existing associations,
+    # and corresponding inclusion hashes.
+    # @param [Hash] includes
+    # @return [Array] an array of pairs [association, include_hash] for matching associations
+    def expand_includes(includes)
```

Comment:
Having this function just return a hash would force an other lookup of the associations down the road.